### ISSUE TITLE
How to show more than 3 categories on the homepage template?

### ISSUE THREAD

**Issue by orschiro** [woocommerce/storefront#2144]
By default, the theme displays only three product categories on the homepage template?

Can this be changed somehow to show all categories or a certain number (more than 3)?

Thank you!

**Comment by manojsiddoji** [comment]
Yes, We can by Overriding the theme template, with the help of Child theme of the parent theme.
